Add 2 performance testing levels to the AutomatedTesting project

rgba16f opened the pull request:
First a level with approximately 10K cubes with various PBR materials
Second a level with a vegetation system set up to spawn between 10K - 30K instances of cubes with various PBR materials
Plus ancillary data files these levels need.

Signed-off-by: rgba16f <82187279+rgba16f@users.noreply.github.com>
![image](https://user-images.githubusercontent.com/82187279/146461337-099f8587-c4e2-4083-b2ea-7e5b0594b0c2.png)
![image](https://user-images.githubusercontent.com/82187279/146461435-29e2906f-6a20-4b4a-ac14-5f36f2ee9980.png)

smurly reviewed (approved):
there's been a discussion about putting graphics related levels in a subfolder Levels/Graphics/ or possibly if there will be more performance test levels we could have Levels/Performance/
moving later is possible, but if you can put these levels in Levels/Graphics/ now that might be good. we share this project with others and should keep the root of Levels somewhat organized

nemerle commented:
May I humbly suggest also adding ?another? level where all of the cubes are rotating? ( might be a good test of animation->transform update performance)

rgba16f commented:
Updated the level files to be under AutomatedTesting/Levels/Performance and the prefabs to be under AutomatedTesting/Prefabs/TestData/Graphics/.

Also fixed a data error in the 10KEntities level where 2 prefabs were in the same location.
